debian/control

```diff
@@ -10,8 +10,8 @@ Build-Depends:
  libncurses-dev,
  libpcre3-dev,
  pkg-config,
- python-docutils,
- python-sphinx,
+ python3-docutils,
```

Comment:
This line is superfluous for Debian and Ubuntu.

The python3-sphinx package already depend on python3-docutils on the currently supported versions of Ubuntu og Debian.

Reply:
I'm fine with pruning one dependency, but it's here mainly because it's an explicit requirement (rst2man) and not a byproduct of using sphinx.